tests/setup/odh-core.yaml

```diff
@@ -1,77 +1,33 @@
-# This file should always match the KfDef used to deploy ODH Core components
 apiVersion: kfdef.apps.kubeflow.org/v1
 kind: KfDef
 metadata:
-  name: odh-core
+  name: odh-modelmesh
 spec:
   applications:
-  - kustomizeConfig:
-      repoRef:
-        name: manifests
-        path: odh-common
-    name: odh-common
-  - kustomizeConfig:
-      parameters:
-        - name: namespace
-          value: openshift-operators
-      repoRef:
-        name: manifests
-        path: openshift-pipelines/cluster
-    name: openshift-pipelines
-  - kustomizeConfig:
-      repoRef:
-        name: manifests
-        path: odh-dashboard
-    name: odh-dashboard
-  - kustomizeConfig:
-      repoRef:
-        name: manifests
-        path: prometheus/cluster
-    name: prometheus-cluster
-  - kustomizeConfig:
-      repoRef:
-        name: manifests
-        path: prometheus/operator
-    name: prometheus-operator
-  - kustomizeConfig:
-      repoRef:
-        name: manifests
-        path: grafana/cluster
-    name: grafana-cluster
-  - kustomizeConfig:
-      repoRef:
-        name: manifests
-        path: grafana/grafana
-    name: grafana-instance
-  - kustomizeConfig:
-      repoRef:
-        name: manifests
-        path: odh-notebook-controller
-    name: odh-notebook-controller
-  - kustomizeConfig:
-      repoRef:
-        name: manifests
-        path: notebook-images
-    name: notebook-images
-  - kustomizeConfig:
-      overlays:
-        - odh-model-controller
-      repoRef:
-        name: manifests
-        path: model-mesh
-    name: model-mesh
-  - kustomizeConfig:
-      overlays:
-        - metadata-store-mariadb
-        - ds-pipeline-ui
-        - object-store-minio
-        - default-configs
-      repoRef:
-        name: manifests
-        path: data-science-pipelines
-    name: data-science-pipelines
+    - kustomizeConfig:
```

Comment:
Please restore the original test kfdef and insert the new modelmesh `kustomizeConfig`.  
After reviewing the CI test run failure and testing this locally, it is braking the integration tests since no other components are installed and their tests will fail.

Reply:
done